pytentiostat/config_reader.py

```diff
@@ -64,13 +101,53 @@ def get_lsv_params(config_data):
 
 
 def get_ca_params(config_data):
+    """
+    Returns the parameters for a chronoamperometry experiment from the config data.
+
+    Parameters
+    ----------
+    config_data : dict
+        the configuration data
+
+    Returns
+    -------
+    voltage : float
+        The voltage which will be held at in volts.
```

Comment:
can we find a better way to say it maybe?

Reply:
Yeah I changed to make more descriptive.